### PR TITLE
added formatResponse option to content-api

### PR DESCRIPTION
**Pull request by Robogeek95**
Added formatResponse option to the ghost content API,
A fix for "Add an option to skip response formatting in Content/Admin API SDKs #73"
#73

The contentAPI will now have the "format response" boolean property, as an option
like this:
const api = new GhostContentAPI({
url: 'YOUR URL',
key: 'YOUR KEY',
version: "VERSION",
formatResponse: true/false
})

### CODE REVIEW

File: packages/content-api/lib/index.js [hunk 1]
```diff
@@ -86,13 +86,16 @@ export default function GhostContentAPI({url, host, ghostPath = 'ghost', version
...
+                if (res.data[resourceType].length === 1 && !res.data.meta) {
+                    return res.data[resourceType][0];
+                }
+                return Object.assign(res.data[resourceType], { meta: res.data.meta });
             }
-            if (res.data[resourceType].length === 1 && !res.data.meta) {
-                return res.data[resourceType][0];
-            }
-            return Object.assign(res.data[resourceType], {meta: res.data.meta});
+            return res;
```
Comment (nextwebb) [inline packages/content-api/lib/index.js:98]: Why are you not returning res.data but res ?? In this case

File: packages/content-api/lib/index.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ import axios from 'axios';
 const supportedVersions = ['v2', 'v3', 'canary'];
 const name = '@tryghost/content-api';
 
-export default function GhostContentAPI({url, host, ghostPath = 'ghost', version, key}) {
+export default function GhostContentAPI({ url, host, ghostPath = 'ghost', version, key, formatResponse = true }) {
```
Comment (nextwebb) [inline packages/content-api/lib/index.js:6]: Setting the formatResponse to default. Seems okay to me.

Comment (Robogeek95) [reply packages/content-api/lib/index.js:6]: the response should be formatted unless i specify it shouldn't by setting formatResponse option to false

File: packages/content-api/lib/index.js (hunk 1 above)
Comment (Robogeek95) [reply packages/content-api/lib/index.js:98]: i feel that returns the raw response data without any formatting

Comment (nextwebb) [reply packages/content-api/lib/index.js:98]: Have you tested it that it returns data without formatting it??

Comment (phawazzzy) [reply packages/content-api/lib/index.js:98]: It is returning res because that is what is in the content API while that of admin API is data

**Comment by simplytunde** [comment]
@Robogeek95 were you able test this?

**Comment by Robogeek95** [comment]
> @Robogeek95 were you able test this?

Yes, i logged the returned data to the console for both options for the formatResponse

**Pull request merged**